src/System.Management.Automation/engine/lang/parserutils.cs

```diff
@@ -1521,6 +1556,81 @@ public bool MoveNext()
             return true;
         }
     }
+
+    /// <summary>
+    /// The simple enumerator class is used for the range operator '..'
+    /// in expressions like 'A'..'B' | ForEach-Object { $_ }
+    /// </summary>
+    internal class CharRangeEnumerator : IEnumerator
+    {
+        private int _increment = 1;
+
+        private bool _firstElement = true;
+
+        public CharRangeEnumerator(char lowerBound, char upperBound)
```

Comment:
This should probably go with the other methods in the class.

Reply:
I use your link https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1201.md
 - constructors is after fields.

Reply:
Ah I missed that this was the constructor... My bad